showcase/src/Main.elm

```diff
@@ -18,6 +18,7 @@ import Sizes
 import Tables.Stories as Tables
 import TextField
 import Texts
+import UI.Checkbox as Checkbox
```

Comment:
is this being used?

Reply:
No, it's just here so we know when compilations fail. There isn't yet a single use-case for it, just the component on Design Core System's Figma.